int32 id
geometry_msgs/Point center
geometry_msgs/Vector3 direction
geometry_msgs/Vector3 velocity
float32 depth_max
float32 depth_min
float32 bottom
float32 top
float32 right
float32 left
float32 width
float32 height
float32 upper_angle
float32 lower_angle
bool dynamic
bool cooperative
bool collision_warning
string fusion 
